src/main/resources/stairway/db/changesets/20210416_flightworking.yaml

```diff
@@ -0,0 +1,49 @@
+databaseChangeLog:
+  - changeSet:
+      id: flightworking
+      author: JCz
+      changes:
+        - dropNotNullConstraint:
+            tableName: flightlog
+            columnName: working_parameters
+            remarks: deprecating for flightworking table
+        - addColumn:
+            tableName: flightlog
+            columns:
+              - column:
+                  name: id
+                  type: uuid
+                  constraints:
+                    unique: true
+        - createTable:
+            tableName: flightworking
+            columns:
+              - column:
+                  name: flightlog_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+              - column:
+                  name: key
+                  type: text
+                  constraints:
+                    nullable: false
+              - column:
+                  name: value
+                  type: text
+                  remarks: serialized object representation
+                  constraints:
+                    nullable: false
+        - addPrimaryKey:
+            tableName: flightworking
+            columnNames: flightlog_id, key
+            constraintName: pk_flightworking
+        - addForeignKeyConstraint:
+            baseTableName: flightworking
+            baseColumnNames: flightlog_id
+            referencedTableName: flightlog
+            referencedColumnNames: id
```

Comment:
Interested in thoughts on cascaded deletes (I wrote delete logic in `FlightDao::deleteInner()` to handle this).

Reply:
I don't have any objections to cascaded deletes, but I also don't have much practical experience with them.

Reply:
Either way works for me. Given that we are asking the DB to do the constraint checking, it makes sense to me to let the DB do the cascading as well.